schema_utils.py

```diff
@@ -465,6 +465,19 @@ def is_at_most(obj, max_value):
         """
         return obj <= max_value
 
+    @staticmethod
+    def is_length_atmost(obj, max_value):
```

Comment:
IS it same as `has_length_at_most`? In general, try to find existing validators and only add new validator if something doesn't exist*

Reply:
Done.